cclib/parser/turbomoleparser.py

```diff
@@ -66,6 +66,25 @@ def normalisesym(self, label):
     def before_parsing(self):
         self.geoopt = False # Is this a GeoOpt? Needed for SCF targets/values.
 
+    def split_molines(self, inline):

```

Comment:
This should be a static method: https://stackoverflow.com/q/12179271/3249688

Reply:
Agreed!

Reply:
Better, just make it a module-level function.